README.md

```diff
@@ -55,6 +55,10 @@ timesync_dhcp_ntp_servers: no
 # Zero threshold disables all steps.
 timesync_step_threshold: 1.0
 
+# Max Distance to accept updates from timeservers
```

Comment:
Can you please expand it to "Maximum root distance to accept measurements for NTP servers" ?

Reply:
Done!

Reply:
I made a typo here. I meant "measurements from NTP servers".

Reply:
Missed that too.. Fixed! :-)